scripts/Vote.ts

```diff
@@ -1,10 +1,53 @@
+import * as dotenv from "dotenv";
+import { attachToBallot, configureWallet, getArguments } from "./Utils";
+import { ethers } from "ethers";
+import { util } from "chai";
+import { Ballot } from "../typechain-types";
+dotenv.config();
 
-async function voteScript() {
-    console.log("")
-    }
-    
-    voteScript().catch((error) => {
-        console.error(error);
-        process.exitCode = 1;
-    });
-    
\ No newline at end of file
+async function voteScript(): Promise<void> {
+  let action: string = "Voting";
+
+  // Grab the proposal from the user
+  // If a user tries to send more than 1 proposal number throw an error
+  // otherwise grab the value of the argument
+  let arg: any = getArguments(process.argv);
+  if (arg.length > 2) {
+    throw new Error("Please only vote for one proposal");
+  } else {
+    arg = arg[0];
+  }
+
+  console.log(`You are voting for proposal #${arg}`);
+
+  // Create a signer for the voter
+  const signerWallet: ethers.Wallet = configureWallet(
+    process.env.VOTER_PRIVATE_KEY
+  );
+
+  // Attach signer/voter to the contract
+  const voterWithContract: Ballot = await attachToBallot(signerWallet);
+
+  // Cast a vote as the voter and the argument passed into the script
+  const voteCasted = await voterWithContract.vote(arg);
```

Comment:
Here we need to be careful. An example:
Suppose you pass 1 as arg. You parse it and send that 1 to the contract.

Contract will do this: 
` proposals[arg].voteCount += sender.weight;`

Where proposals[arg] = proposals[1] is equal to "Proposal 2" because of zero indexing. 
To cast a vote to proposal 1 we need to proposals[arg - 1] = proposals[0]. That would be equal to "Proposal 1"